infrastructure/scripts/execution-plan/plan-pr.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 set -ueo pipefail
+SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]:-$0}"; )" &> /dev/null && pwd 2> /dev/null; )";
 
 main_branch="main"
 current_branch="$(git rev-parse --abbrev-ref HEAD)"
 # When called locally with Makefile, the default parameters are used
 base="${1:-$(git merge-base "${main_branch}" "${current_branch}")}"
 head="${2:-$(git rev-parse HEAD)}"
 
-git diff --diff-filter=ACMRT --name-only "$base" "$head" | docker run -i -v "$(pwd)":/repo eu.gcr.io/freiheit-core/images/execution-plan:2.0-scratch-NG-8 build-pr
+git diff --diff-filter=ACMRT --name-only "$base" "$head" | docker run -i -v "$(pwd)":/repo $($SCRIPT_DIR/container.sh) build-pr
```

Comment:
You would need to quote `$SCRIPT_DIR` in case it contains whitespace. And for improved readability you could use the `${SCRIPT_DIR}` form.